server/proxy/freerdp_proxy.c

```diff
@@ -48,40 +49,62 @@ static void cleanup_handler(int signum)
 	pf_server_free(server);
 
 	WLog_INFO(TAG, "exiting.");
-	exit(signum);
+	exit(0);
 }
 
-int main(int argc, char* argv[])
+static void pf_server_register_signal_handlers()
```

Comment:
missing `(void)`, defaults to arbitrary arguments in `C` (as opposed to `C++`)

Reply:
missed it, thanks